### DISCUSSION TITLE
Sort strings by "last updated"

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#9212] [Q&A] [answered]
When looking into the list of strings I would like to sort them by updated or modified date.

The background is I would like to see the latest actievis in a language. Based on the last commits I can see that someone did translations in a specific language but I'm not able to see what was translated.

**Comment by Geeyun-JY3** [comment]
I think you should file an issue for feature request instead of starting a discussion.

**Reply by buhtz** [reply]
So you sure this isn't implemented yet? I assumed I just missed an option in the UI. If it is the case the maintainers can easily convert a discussion into an issue on Microsoft GitHub.

**Reply by Geeyun-JY3** [reply]
> So you sure this isn't implemented yet? I assumed I just missed an option in the UI.

Obviously, there is no such option in the UI. 
<img src="https://user-images.githubusercontent.com/49396421/237002943-0d4944d8-4caf-47d7-83d9-18d4afa51070.png" width="25%" height="25%">

> If it is the case the maintainers can easily convert a discussion into an issue on Microsoft GitHub.

Actually, the maintainers can only create a new issue from the discussion now. 
Technically, it is not a conversion: https://github.com/orgs/community/discussions/2861#discussioncomment-696235

**Comment by buhtz** [comment, marked as answer]
#9215
